test/specs/PublicMethodsSpec.js

```diff
@@ -246,6 +246,14 @@ describe("Public Method Tests", function() {
         expect(sliderValue).toBe(valueToSet);
       });
 
+      it("properly sets the value of the slider when given a string value", function(){
```

Comment:
could you also add a test for when you pass a string value that cannot be properly converted into a number?

For example: 

```js
testSlider.slider('setValue', "rolo");
```